packages/desktop/src/widget_factory.rs

```diff
@@ -320,6 +345,9 @@ impl WidgetFactory {
       .get_webview_window(widget_id)
       .context("No Tauri window found for the given widget ID.")?;
 
+    #[cfg(target_os = "windows")]
+    let _ = window.as_ref().window().deallocate_app_bar();
+
```

Comment:
Also we'd need to clean up appbar space on exit (e.g. via systray). something like this should work (modify [`main.rs` here](https://github.com/glzr-io/zebar/blob/e394f9d7f5968224103798bfa258b92389134dd2/packages/desktop/src/main.rs#L94)):

```rust
  app.run(|app, event| {
    if let RunEvent::ExitRequested { code, api, .. } = &event {
      if code.is_none() {
        // Keep the message loop running even if all windows are closed.
        api.prevent_exit();
      } else {
        task::block_in_place(|| {
          block_on(async move {
            let widget_factory = app.state::<Arc<WidgetFactory>>();
            let widget_states = widget_factory.states().await;
            println!("{:?}", widget_states);

            // todo: loop over appbar widgets and deallocate appbar space
          })
        });
      }
    }
  });
```